### PR TITLE
tidb-lightning: add lightning physical import mode usage

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Translate part of the docs from the TiDB Lightning refactoring plan (https://github.com/pingcap/docs-cn/pull/9653), including two docs:
- `tidb-lightning/tidb-lightning-physical-import-mode-usage.md`
- `tidb-lightning/tidb-lightning-physical-import-mode.md`

Some paragraphs are taken from https://github.com/pingcap/docs/blob/master/tidb-lightning/tidb-lightning-backends.md

**Note: Some v6.2 features rely on these two docs, so we need to merge them ASAP.**

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9653, https://github.com/pingcap/docs-cn/pull/10428
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+# to slow down. For different storage mediums, this parameter may need to be
+# adjusted to achieve the best performance.
+io-concurrency = 5
+```
+
+During the import, each table is split into one "index engine" to store indices, and multiple "data engines" to store row data.
+
+`index-concurrency` controls the maximum concurrency of the index engine. When you adjust `index-concurrency`, you need to ensure that `index-concurrency * the number of source files of each table > region-concurrency` to ensure that the cpu is fully utilized. The ratio is usually between 1.5 ~ 2. Do not set `index-concurrency` too high and not lower than 2 (default). Too high `index-concurrency` causes too many pipelines to be built, which causes the index-engine import stage to pile up.
+
+The same goes for `table-concurrency`. Ensure that `table-concurrency * the number of source files of each table > region-concurrency` to ensure that the cpu is fully utilized. A recommended value is around `region-concurrency * 4 / the number of source files of each table` and not lower than 4.
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:161]:
```suggestion
`index-concurrency` controls the maximum concurrency of the index engine. When you adjust `index-concurrency`, make ensure that `index-concurrency * the number of source files of each table > region-concurrency` to ensure that the CPU is fully utilized. The ratio is usually between 1.5 ~ 2. Do not set `index-concurrency` too high and not lower than 2 (default). Too high `index-concurrency` causes too many pipelines to be built, which causes the index-engine import stage to pile up.

The same goes for `table-concurrency`. Make sure that `table-concurrency * the number of source files of each table > region-concurrency` to ensure that the CPU is fully utilized. A recommended value is around `region-concurrency * 4 / the number of source files of each table` and not lower than 4.
```

File: tidb-lightning/tidb-lightning-physical-import-mode-usage.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+If the table is large, Lightning will split the table into multiple batches of 100 GiB. The concurrency is controlled by `table-concurrency`.
+
+`index-concurrency` and `table-concurrency` has little effect on the import speed. You can leave them in the default value.
+
+`io-concurrency` controls the concurrency of file read. The default value is 5. At any given time, only 5 handles are performing read operations. Because the file read speed is usually not a bottleneck, you can leave this configuration in the default value.
+
+After the file data is read, Lightning needs to do some post-processing, such as encoding and sorting the data locally. The concurrency of these operations is controlled by `region-concurrency`. The default value is the number of CPUs. You can leave this configuration in the default value. It is recommended to deploy Lightning on a separate servers from other components. If you must deploy Lightning with other components, you may need to lower `region-concurrency` according to the load.
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode-usage.md:169]:
```suggestion
After the file data is read, Lightning needs to do some post-processing, such as encoding and sorting the data locally. The concurrency of these operations is controlled by `region-concurrency`. The default value is the number of CPUs. You can leave this configuration in the default value. It is recommended to deploy Lightning on a separate server from other components. If you must deploy Lightning together with other components, you need to lower `region-concurrency` according to the load.
```

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+title: Physical Import Mode
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:2]:
```suggestion
title: Physical Import Mode
summary: Learn about the physical import mode in TiDB Lightning.
```

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+---
+
+# Physical Import Mode
+
+Physical import mode is an efficient and fast import mode that inserts data directly into TiKV ndoes as key-value pairs without going through the SQL interface. It is suitable for importing up to 100 TB of data.
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:7]:
```suggestion
Physical import mode is an efficient and fast import mode that inserts data directly into TiKV nodes as key-value pairs without going through the SQL interface. It is suitable for importing up to 100 TB of data.
```

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Before you use physical import mode, make sure to read [Requirements and restrictions](#requirements-and-restrictions).
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:9]:
```suggestion
Before you use the physical import mode, make sure to read [Requirements and restrictions](#requirements-and-restrictions).
```

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+## Implementation
+
+1. Before importing data, TiDB Lightning automatically switches the TiKV nodes to "import mode", which improves write performance and stops PD scheduling and auto-compaction.
+
+2. `tidb-lightning` creates table schemas in the target database and fetches the metadata.
+
+3. Each table is divided into multiple contiguous **blocks**, so that Lightning can import data data from large tables (200 GB+) in parallel.
+
+4. `tidb-lightning` prepares a "engine file" for each block to handle key-value pairs. `tidb-lightning` reads the SQL dump in parallel, converts the data source to key-value pairs in the same encoding as TiDB, sorts the key-value pairs and writes them to a local temporary storage file.
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:19]:
```suggestion
4. `tidb-lightning` prepares an "engine file" for each block to handle key-value pairs. `tidb-lightning` reads the SQL dump in parallel, converts the data source to key-value pairs in the same encoding as TiDB, sorts the key-value pairs and writes them to a local temporary storage file.
```

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+6. After all engine files are imported, `tidb-lightning` compares the checksum between the local data source and the downstream cluster, and ensures that the imported data is not corrupted. Then `tidb-lightning` analyzes the new data (`ANALYZE`) to optimize the future operations. Meanwhile, `tidb-lightning` adjusts the `AUTO_INCREMENT` value to prevent conflicts in the future.
+
+    The auto-increment ID is estimated by the **upper bound** of the number of rows, and is proportional to the total size of the table data file. Therefore, the auto-increment ID is usually larger than the actual number of rows. This is normal because the auto-increment ID [is not necessarily contiguous](/mysql-compatibility.md#auto-increment-id).
+
+7. After all steps are completed, `tidb-lightning` automatically switches the TiKV nodes to "normal mode", and the TiDB cluster can provide services normally.
+
+## Requirements and restrictions
+
+### 运行环境需求
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:33]:
```suggestion
```

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+# Physical Import Mode
+
+Physical import mode is an efficient and fast import mode that inserts data directly into TiKV ndoes as key-value pairs without going through the SQL interface. It is suitable for importing up to 100 TB of data.
+
+Before you use physical import mode, make sure to read [Requirements and restrictions](#requirements-and-restrictions).
+
+## Implementation
+
+1. Before importing data, TiDB Lightning automatically switches the TiKV nodes to "import mode", which improves write performance and stops PD scheduling and auto-compaction.
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:13]: For this step, the latest version is https://github.com/pingcap/docs-cn/blob/master/tidb-lightning/tidb-lightning-physical-import-mode.md#%E5%8E%9F%E7%90%86%E8%AF%B4%E6%98%8E for v6.2. The updates in https://github.com/pingcap/docs-cn/pull/10522 will be aligned in another PR, right?

Comment (ran-huang) [reply tidb-lightning/tidb-lightning-physical-import-mode.md:13]: Yes. It will be aligned in a v6.2 PR.

**Review by lilin90 — APPROVED** [review]
LGTM

**Review by sunzhaoyang — APPROVED** [review]
LGTM
